common/src/main/java/org/apache/cassandra/sidecar/common/data/QualifiedTableName.java

```diff
@@ -1,12 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.apache.cassandra.sidecar.common.data;
 
+import com.google.inject.Inject;
 import org.apache.cassandra.sidecar.common.utils.ValidationUtils;
 
 /**
  * Contains the keyspace and table name in Cassandra
  */
 public class QualifiedTableName
 {
+    @Inject
+    static ValidationUtils validationUtils;
```

Comment:
Nit: since rest of the utils class have static methods, to distinguish this class, shall we name it something like `ParamsValidator` ?

Reply:
Since this class does very Cassandra-specific validations, how does `CassandraInputValidator` sound?